.sync.yml

```diff
@@ -0,0 +1,34 @@
+---
+
+mock_with: ':rspec'
+

```

Comment:
could you add a few items here:
https://github.com/MiamiOH/puppet-duo_authproxy/blob/master/.sync.yml#L1

the gitlab and travis